config/install/node.type.title.yml

```diff
@@ -0,0 +1,16 @@
+langcode: en
```

Comment:
This file can be removed. It refers to the node type "title" which we need to remove in favor of "people".